Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    xcres (0.4.4)
+    xcres (0.5)
```

Comment:
This is left from the version bump. `bundle install; git add Gemfile.lock; git commit --amend` should do the job.